content/articles/ordering-a-wildcard-certificate.markdown

```diff
@@ -0,0 +1,47 @@
+---
+title: Ordering a Wildcard SSL Certificate
+excerpt: How to order a wildcard SSL certificate with DNSimple.
+categories:
+- SSL Certificates
+---
+
+# Ordering a Standard SSL Certificate
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Wildcard certificates allow you to secure all subdomains of a domain with a single certificate. Wildcard certificates can make certificate management easier in some cases. We still recommend non-wildcard certificates for most use cases.
+
+Wildcard certificates are no different to standard certificates, they only contain an `*` as the subdomain, so please make sure to check out the article about [Ordering a Standard Certificate](/articles/ordering-standard-certificate/) too.
+
+
+## Let's Encrypt or Standard
+
+Both providers support wildcard certificates Let's Encrypt and Comodo. Let's Encrypt certificates are free but are only valid for 60 days, Comodo certificates are valid for one year or longer but also cost $100 per wildcard certificate.
```

Comment:
> Both providers support wildcard certificates Let's Encrypt and Comodo.

This sentence is missing something. Is there a typo?

Reply:
> Let's Encrypt certificates are free but are only valid for 60 days

This is incorrect. They are valid for 90 days.

Generally speaking, I think we should avoid to repeat information that are already available in the specific article, and are even subject to become outdated (e.g. the price, or the duration).

You can stay more generic, and explain that we offer wildcard certificates via both Let's Encrypt and Standard certificates. To learn more about the differences then you can refer to the article.